translations/frontend/en.json

```diff
@@ -2429,6 +2429,10 @@
                                 "title": "Entity not tracked"
                             }
                         }
+                    },
+                    "compare":{
+                        "alert_info_1": "You are comparing the period",
```

Comment:
I don't think we should have the word "alert" in the key since in the future we might display the text within another element.

Reply:
well yes maybe it's better `ui.panel.energy.compare.info_1` `ui.panel.energy.compare.info_2`